drivers/sensor/grow_r502a/grow_r502a.h

```diff
@@ -182,7 +171,12 @@ struct grow_r502a_data {
 #endif /* CONFIG_GROW_R502A_TRIGGER */
 
 	struct r502a_buf tx_buf;
-	struct r502a_buf rx_buf;
+
+	struct r502a_buf rx_data_buf;
+	union r502a_packet rx_hdr_pkt;
+	int rx_len;
+	bool rx_header;
```

Comment:
i repeat remove rx-header... first do self review and remove unnecessary code

Reply:
Okay will do that, thanks for the review.